daemon/lxd.go

```diff
@@ -166,3 +169,75 @@ func (l *LXD) DeleteContainer(hostname string) (bool, error) {
 
 	return true, nil
 }
+
+func (l *LXD) CreateContainerBootstrapFile(container pfmodel.Container, fullPath string) error {
+	var content string
+	mode := 644
+	contentType := "file"
+	writeMode := "overwrite"
+
+	for _, bs := range container.Bootstrappers {
+		content, mode = util.SetupBootstrapFileContent(bs, content, mode)
+	}
+
+	bootstrapFile, err := util.WriteStringToFile(fullPath, content)
+	if err != nil {
+		return err
+	}
+
+	// Setup the various options for a container file upload
+	fileArgs := client.ContainerFileArgs{
+		Content:   bootstrapFile,
+		UID:       0,
+		GID:       0,
+		Mode:      mode,
+		Type:      contentType,
+		WriteMode: writeMode,
+	}
+
+	err = l.targetSrv.CreateContainerFile(container.Hostname, fullPath, fileArgs)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *LXD) ExecContainerBootstrap(container pfmodel.Container, fullPath string) (bool, error) {
```

Comment:
can you rename `fullPath` it is not showing intent
this is the path of bootstrap file right?